src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -253,6 +254,40 @@ export default function EditSidebar({
                       {mediaTranslations.btn_save_file}
                     </button>
                   )}
+     
+                  <button
```

Comment:
I think it makes sense to define this as `<label for="replace-file-input" ...>` instead of a button with a custom onclick property.